packages/e2e-tests/specs/rich-text.test.js

```diff
@@ -57,6 +62,21 @@ describe( 'RichText', () => {
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
 
+	it( 'should return focus when pressing formatting button', async () => {
+		await clickBlockAppender();
+		await page.keyboard.type( 'Some ' );
+		// Sets isTyping to false.
+		await moveMouse();
+		await page.click( '[aria-label="Bold"]' );
+		await page.keyboard.type( 'bold' );
+		// Sets isTyping to false.
```

Comment:
We should do `page.keyboard.press( 'Escape' )` instead. It should work just as well, and is simpler than imitating some "shake the mouse" interaction.

Reply:
Oh, thanks, didn't realise.